main.css

```diff
@@ -163,5 +169,28 @@ img {
 
 .cards-group-2 .card {
   flex-grow: 1;
-  /* flex-basis: 100rem; */
+}
+
+/* Image Component */
+.image-container {
+  display: flex;
```

Comment:
you are creating things flex-centered again again in different classes 

to reduce the code length you can use a helper class for flex-center and then use it everywhere you want flex center

Reply:
Yeah, Noted! Will keep that in mind while refactoring.